packages/block-editor/src/components/create-new-post-link/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import { addQueryArgs } from '@wordpress/url';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { store as blockEditorStore } from '../../store';
+
+const CreateNewPostLink = ( { clientId } ) => {
+	const postType = useSelect( ( select ) => {
+		const { getBlockAttributes } = select( blockEditorStore );
+		return getBlockAttributes( clientId )?.query?.postType;
+	} );
```

Comment:
Looks like a missing dependency array:
```suggestion
	const postType = useSelect( ( select ) => {
		const { getBlockAttributes } = select( blockEditorStore );
		return getBlockAttributes( clientId )?.query?.postType;
	}, [ clientId ] );
```

Reply:
Thanks, fixed in e51184ce8d so I could adjust formatting to satisfy the linter at the same time.